doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md

```diff
@@ -1,4 +1,6 @@
 #<a name="top"></a>NGSIGroupingInterceptor
+**IMPORTANT NOTE: from release 1.6.0, this feature is deprecated in favour of Name Mappings.**
```

Comment:
A link to the deprecated_and_removed.md file, to the place were this deprecation is explained, could help.